azdev/commands.py

```diff
@@ -34,6 +34,10 @@ def operation_group(name):
     with CommandGroup(self, 'verify', operation_group('pypi')) as g:
         g.command('history', 'check_history')
 
+    with CommandGroup(self, 'break-change', operation_group('break_change')) as g:
```

Comment:
I think `breaking-change` is more correct. But if your concern is it's too long, you can use `change` or `command-change` as well. If you use `command-change` then you can remote `command-` prefix in the sub names

Reply:
Please also provide the help description for those commands and command groups.